deploy/crds/metal3.io_baremetalhosts_crd.yaml

```diff
@@ -63,6 +63,26 @@ spec:
         spec:
           description: BareMetalHostSpec defines the desired state of BareMetalHost
           properties:
+            bios:
+              description: BIOS settings for bare metal server
+              properties:
+                bootMode:
```

Comment:
So I was thinking in reusing that to set the capability at the node. But what about if we do not want to configure bios? I guess this bios section just needs to be present when you want to do automated bios management. So what do we do if not? @hardys @dhellmann

Reply:
The thread I linked is below: https://github.com/metal3-io/baremetal-operator/pull/302/files#r334923564

The consensus is not have this field under bios at all.